src/Pim/Bundle/CatalogBundle/Elasticsearch/Sorter/BaseSorter.php

```diff
@@ -55,25 +55,14 @@ public function addFieldSorter($field, $direction, $locale = null, $scope = null
 
         switch ($direction) {
             case Directions::ASCENDING:
-                $sortClause = [
-                    $field => [
-                        "order" => $direction,
-                        "missing" => "_last"
-                    ]
-                ];
-                $this->searchQueryBuilder->addSort($sortClause);
-
-                break;
             case Directions::DESCENDING:
                 $sortClause = [
                     $field => [
-                        "order" => $direction,
```

Comment:
simple quotes please (If I merge the metrics PR before yours, I also fixed that up)